### DISCUSSION TITLE
No fastg output

### DISCUSSION THREAD

**Discussion by eperezv** [ablab/spades#760] [Q&A] [answered]
Hello,

I am running metaplasmidSPAdes and I get not fastg files. This happened with v. 3.14.1 and is the same with 3.15. I have no problem when running the same command (including paired fastq files with --meta) without --plasmid.

Is this normal? Is there any easy way to generate the fastg files once the assembly is finished.

Thank you

**Comment by asl** [comment, marked as answer]
Hello

Due to way how metaplasmidSPAdes works (see https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6581055/ for more information) there is effectively no assembly graph, hence no FASTG / GFA output.
